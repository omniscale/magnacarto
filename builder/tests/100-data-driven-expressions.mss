#labels_field {
    text-name: [name];
    text-size: "2+6";
}

#labels_field2 {
    text-name: [name];
    text-size: "[size]";
}

#labels_roads_refs {
  [type='motorway'] {
    shield-name: "[ref]";
    shield-size: "[size]";
    shield-face-name: "DejaVu Sans Book";
    shield-fill: "[color]";
    shield-file: url("img/rail-[width].svg");
    shield-placement: line;
  }
}
