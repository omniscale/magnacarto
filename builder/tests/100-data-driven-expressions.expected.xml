<Map srs="epsg:3857">
  <Parameters></Parameters>
  <FontSet name="fontset-1">
    <Font face-name="DejaVu Sans Book"></Font>
  </FontSet>
  <Style name="labels_field" filter-mode="first">
    <Rule>
      <TextSymbolizer size="2+6">[name]</TextSymbolizer>
    </Rule>
  </Style>
  <Style name="labels_field2" filter-mode="first">
    <Rule>
      <TextSymbolizer size="[size]">[name]</TextSymbolizer>
    </Rule>
  </Style>
  <Style name="labels_roads_refs" filter-mode="first">
    <Rule>
      <Filter>([type] = &#39;motorway&#39;)</Filter>
      <ShieldSymbolizer file="img/rail-[width].svg" fill="[color]" fontset-name="fontset-1" placement="line" size="[size]">[ref]</ShieldSymbolizer>
    </Rule>
  </Style>
  <Layer name="labels_field" srs="" status="off">
    <StyleName>labels_field</StyleName>
  </Layer>
  <Layer name="labels_field2" srs="" status="off">
    <StyleName>labels_field2</StyleName>
  </Layer>
  <Layer name="labels_roads_refs" srs="" status="off">
    <StyleName>labels_roads_refs</StyleName>
  </Layer>
</Map>